konig-schemagen/src/main/java/io/konig/schemagen/java/JavaClassBuilder.java

```diff
@@ -62,6 +61,7 @@
 	private OwlReasoner reasoner;
 	private Graph graph;
 	private TypeInfo owlThing;
+	private ShapeHandler shapeHandler;

```

Comment:
Why are you making changes to JavaClassBuilder?  This does not seem to having anything to do with Spanner tables.